src/views/dialogs/WithdrawDialog2/WithdrawStatus.tsx

```diff
@@ -0,0 +1,79 @@
+import { useMemo } from 'react';
+
+import { ButtonAction } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useParameterizedSelector } from '@/hooks/useParameterizedSelector';
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import { AssetIcon } from '@/components/AssetIcon';
+import { Button } from '@/components/Button';
+import { Icon, IconName } from '@/components/Icon';
+import { LoadingSpinner } from '@/components/Loading/LoadingSpinner';
+import { Output, OutputType } from '@/components/Output';
+
+import { selectWithdraw } from '@/state/transfersSelectors';
+
+type WithdrawStatusProps = {
+  txHash: string;
+  chainId: string;
+  onClose: () => void;
+};
+
+// TODO(deposit2.0): localization for this whole component
+export const WithdrawStatus = ({ txHash, chainId, onClose }: WithdrawStatusProps) => {
+  const stringGetter = useStringGetter();
+  const withdraw = useParameterizedSelector(selectWithdraw, txHash, chainId);
+
+  const transferSuccess = withdraw?.status === 'success';
+
+  const statusDescription = useMemo(() => {
+    if (transferSuccess) return 'Your funds have been withdrawn.';
+
+    return 'Your funds will be withdrawn shortly, you may safely close this window.';
```

Comment:
This should be two separate sentences xD

Reply:
😅

Reply:
ill add to follow up